frontend/src/features/review/components/review/ActionButtons.tsx

```
@@ -0,0 +1,23 @@
+const ActionButtons = ({
+  reviewId,
+  handleEditReview,
+}: {
+  reviewId: string;
+  handleEditReview: (index: string) => void;
+}) => (
+  <div className='flex items-center gap-2'>
+    <button className='px-3 py-1.5 rounded-xl'>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)